onnx/backend/test/data/real/test_inception_v2/data.json

```diff
@@ -1 +1 @@
-{"model_name": "inception_v2", "url": "https://s3.amazonaws.com/download.onnx/models/opset_8/inception_v2.tar.gz"}
\ No newline at end of file
+{"model_name": "inception_v2", "url": "https://s3.amazonaws.com/download.onnx/models/inception_v2.tar.gz"}
```

Comment:
Why changing this?

Reply:
this change was made by the `stat-coverage` tool.